dgraph/cmd/zero/zero_test.go

```diff
@@ -112,3 +117,38 @@ func TestProposalKey(t *testing.T) {
 	}
 	require.Equal(t, len(uniqueKeys), 10, "each iteration should create unique key")
 }
+
+func TestZeroHealth(t *testing.T) {
+	client := http.Client{Timeout: 3 * time.Second}
+	u := &url.URL{
+		Scheme: "http",
+		Host:   testutil.ContainerAddr("zero1", 6080),
+		Path:   "health",
+	}
+
+	// JSON format
+	req, err := http.NewRequest("GET", u.String(), nil)
+	require.NoError(t, err)
+
+	req.Header.Add("Accept", `application/json`)
+	resp, err := client.Do(req)
+	require.NoError(t, err)
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	require.NoError(t, err)
+	require.NotEmpty(t, body)
+	require.True(t, json.Valid(body))
```

Comment:
Can you validate the contest of the json body? What all headers are present and their values?